cypress/support/commands.js

```diff
@@ -23,3 +23,19 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (username, password) => { 
+  cy.request({
+    method: 'POST',
+    url: 'https://demoqa.com/Account/v1/Login',
+    body: {
+      userName: username,
+      password
+    }
+  }).then((response) => {
+    cy.setCookie('token', response.body.token)
+    cy.setCookie('userID', response.body.userId)
+    cy.setCookie('expires', response.body.expires)
+    cy.setCookie('userName', response.body.username)
+  })
+ })
```

Comment:
```suggestion
 });
 
```
use `;` where it needed

Reply:
can not see any changes